components/MainButton.js

```diff
@@ -0,0 +1,44 @@
+import styled from '@material-ui/core/styles'
+import PropTypes from 'prop-types'
+import Button from '@material-ui/core/Button'
+import { createTheme } from '@material-ui/core/styles'
+
+//custom theme for coloring
+const theme = createTheme({
+  palette: {
+    primary: {
+      light: 'purple',
+      main: 'purple',
+      dark: 'purple',
+      contrastText: '#fff',
+    },
+    secondary: {
+      light: '#ff7961',
+      main: 'purple',
+      dark: '#ba000d',
+      contrastText: '#000',
+    },
+  },
+})
+
+//receives 'buttonElements' from index.js -- Custom text can be passed using the elements array. //
```

Comment:
Awesome job leaving a comment. Just a minor note, if this button is re-used, the text might come from somewhere else than just `index.js`

Reply:
Going to eliminate the array from now and just pass string props to the buttons to simplify.